### DISCUSSION TITLE
unable to make a selection or edit a selection

### DISCUSSION THREAD

**Discussion by ENDERSTIXgaming** [maruohon/litematica#274] [Q&A]
I hopped on my singleplayer creative world. finished editing something I was working on and was about to make a schem of it but the area wasn't selected anymore even though it was previously and I hadn't messed with it. so I tried making a new selection but I couldn't I tried M + A and pressing new selection button in the menu and I was able to make a selection in the menu and name it but it wasn't appearing in the world so idk what I'm doing wrong and this seems to be the place to ask for help. if it isn't please direct me to where I need to go to get help

**Comment by maruohon** [comment]
For help requests the new Discussion area linked on the top would be more suitable, but Discord would be even easier and "less overhead". I need to link and a mention of both of these in the README and the issue template.

As to the actual problem you are having, I'm assuming you have probably the main rendering disabled (`M + R`)? In that case you would also not see the tool HUD. There is also a separate option in Visuals to just disable the area selection box rendering, but that doesn't have a default hotkey so it's more difficult to accidentally disable that.

Edit: Also another reason for none of the overlays or selections rendering is if you have certain shader packs active in Optifine.

There are also the two separate "sub-modes" to the Area Selection mode, Normal vs. Simple, but if you are creating a new selection via the Area Selection Browser then you would see at least the starting box/corner in either mode. (The Simple mode always has one box that can't be deleted and more can't be added, and thus you don't need the Area Selection Browser at all in the Simple mode. The Normal mode would not render anything if you don't have an active selection in the Area Selection Browser, and the Normal mode allows using multiple sub-regions in the same selection/schematic.)

**Comment by Turtle1637** [comment]
For those who don't know how to use area editor again, the solution is first press M then, where are the area editor mechanics, pick from the three area selection mode, if you got it on normal it doesn't work because the system makes it so you use it as a regular stick even tho you cant break blocks, for you to be able to use the area editor feature again you need to put it back on ¨simple¨ mode. And that's how, you use area editor feature again.

**Comment by sbrl** [comment]
For me, I found I needed to go into the `Area Editor` instead of the `Area Selection browser` to edit the subregions, otherwise it didn't work. 1.21.5.
